doc-content/enterprise-only/springboot/bus-app_con.adoc

```diff
@@ -0,0 +1,25 @@
+[id='bus_app_{context}']
+= {PRODUCT} business application
+
+{PRODUCT} business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example order management or accommodation management. After you create and configure your business application, you can deploy it to an existing service through {CENTRAL} or to the cloud through OpenShift. 
```

Comment:
I would remove the part about deploying through business central as that is not really true. Sin instead of 
you can deploy it to an existing service through {CENTRAL} or to the cloud through OpenShift.

I would say
you can deploy it to an existing server or to the cloud through OpenShift.